components/HeaderNav.js

```diff
@@ -0,0 +1,42 @@
+import { Component } from 'react'
+
+import NavListItems from './NavListItems'
+
+class HeaderNav extends Component {
+  constructor (props) {
+    super(props)
+    this.state = { isOpen: false }
+
+    this.toggleMenu = this.toggleMenu.bind(this)
+  }
+
+  toggleMenu () {
+    this.setState({ isOpen: !this.state.isOpen })
+    if (this.state.isOpen) {
+      document.querySelector("div[data-behavior='myNav']").style.width = '0%'
```

Comment:
I would set these styles and then apply them to the element, rather than manipulating the DOM